rocketpool/watchtower/submit-network-balances.go

```diff
@@ -128,71 +127,44 @@ func (t *submitNetworkBalances) run(state *state.NetworkState) error {
 		return err
 	}
 
+	rp, err := services.GetRocketPool(t.c)
+	if err != nil {
+		return err
+	}
+
 	// Check if balance submission is enabled
 	if !state.NetworkDetails.SubmitBalancesEnabled {
 		return nil
 	}
 
-	// Check the last submission block
-	lastSubmissionBlock := state.NetworkDetails.BalancesBlock.Uint64()
-
 	if state.IsHoustonDeployed {
-		// Get the last balances updated event
-		found, event, err := network.GetBalancesUpdatedEvent(t.rp, lastSubmissionBlock, nil)
-		if err != nil {
-			return fmt.Errorf("error getting event for balances updated on block %d: %w", lastSubmissionBlock, err)
-		}
+		// Check the last submission block
+		lastSubmissionBlock := state.NetworkDetails.PricesBlock
+
+		referenceTimestamp := t.cfg.Smartnode.PriceBalanceSubmissionReferenceTimestamp.Value.(int64)
```

Comment:
can we be defensive here and error out if `refereanceTimestamp == 0`

convert this to time.Time, also